core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairConfiguration.java

```diff
@@ -23,12 +23,14 @@
 public class RepairConfiguration
 {
     public static final double NO_UNWIND = 0.0d;
+    public static final long FULL_REPAIR_SIZE = Long.MAX_VALUE;
```

Comment:
Shouldn't it be assigned to DEFAULT_TARGET_REPAIR_SIZE_IN_BYTES first and then to FULL_REPAIR SIZE?
`public static final long FULL_REPAIR_SIZE = DEFAULT_TARGET_REPAIR_SIZE_IN_BYTES;`

Also, do we need two fields for one being private and one being public? Can't it be the same field?

Reply:
I would like to keep them separate as they are used for different things.
FULL_REPAIR_SIZE is a magic number used to specify "don't use the sub-range repair feature". The default value might change over time and should IMO then be a separate parameter. FULL_REPAIR_SIZE is used for both repair history and range split in order to activate the feature.